js/index.js

```diff
@@ -0,0 +1,77 @@
+  },
+  {
+    imgSrc: './assets/images/audi.jpg',
+    heading: 'Audi RS7 Sport',
+    title: 'Audi RS7 Sport is the latest RS7 to be produced, available in both RHD and LHD.',
+    explainer: 'The Audi RS7 is a largemouth high-performance hatchback sedan with swollen fenders and a spacious interior that’s chock full of the luxuries and high-tech gear you’d expect in a super-sedan.',
+  },
+  {
+    imgSrc: './assets/images/cadilac.jpg',
+    heading: '2020 Cadillac Escalade',
+    title: 'Even though it shares parts with lesser GM products, the 2020 Escalade is unmistakably a luxury SUV.',
+    explainer: 'Along with its chrome-laden shell, the Escalade has a host of upscale standard features such as a leather interior, heated and ventilated front seats, self-parking assist, and more. Every Slade is motivated by a gutsy V-8 that pairs with rear- or all-wheel drive and can tow up to a hefty 8300 pounds.',
+  },
+  {
+    imgSrc: './assets/images/mk8.jpg',
+    heading: 'Volkswagen Golf GTI',
+    title: 'Fears that Volkswagen would annihilate the entire Golf lineup are all but forsaken, with the 2022 Golf GTI set to begin an all-new generation of the sporty hatchback.',
+    explainer: 'The 2022 Golf GTI is all new and marks the storied nameplate\'s eighth generation. While its styling is evolutionary rather than revolutionary, VW has packed the hatchback with more technology and enhanced performance. Its fresh exterior styling is promoted by sleeker LED headlights and a gaping lower grille with fog lights integrated into the mesh.',
+  },
+  {
+    imgSrc: './assets/images/rover.jpg',
+    heading: 'Land Rover Defender',
+    title: 'The 2022 Land Rover Defender might look reminiscent of the 1950s original, but in reality it is a completely modern, high-tech off-roader with unique curb appeal.',
+    explainer: 'Land Rover is introducing a V-8 engine into the Defender lineup for 2022: the 518-hp Supercharged 5.0-liter V-8 that\'s found in the Range Rover and Range Rover Sport models',
+  },
+  {
+    imgSrc: './assets/images/volvo.jpg',
+    heading: '2022 Volvo XC60',
+    title: 'Volvo\'s unique Swedish perspective is evident in every one of the brand\'s products, including its top-seller, the stylish and thoughtfully-designed 2022 XC60 SUV.',
+    explainer: 'Several subtle styling changes help mark the 2022 model year as well, including a tweaked grille, a re-worked front bumper, updated wheel designs, and the introduction of Volvo\'s City Weave cloth upholstery as an option for the first time on the XC60.',
+  },
+];
+
+const featuredHtml = featuredData.map((item, index) => (
+  `
+        <li class="--featured ${index > 1 ? 'd-none' : ''}">
+            <div class="left"><img src="${item.imgSrc}" alt="Featured" class="featured"></div>
```

Comment:
Please add one more property called `alt` for each image in featuredData array and add the alt attribute for each image. As you know, it has a crucial role on the accessibility of your web page, especially for people who have visual impairments.